lib/tasks/proposals.rake

```diff
@@ -0,0 +1,7 @@
+namespace :delete_proposals do
+  desc 'Delete proposals that are older than specified date old'
+  task :expired_proposals => :environment do
+    count = CollectionDraftProposal.delete_all("proposal_status = 'in_work' AND updated_at < '#{30.days.ago}'") if Rails.configuration.proposal_mode
```

Comment:
This selection was verified with the PO, right?

Reply:
Yes, another ticket was written to capture the work for other cases.

Reply:
There is no test for this, which may be hard to do, but I got a demo and I think the logic is good. updated_at < 30 days ago when looking at timestamp would work as expected.